plugins/importexport/medra/classes/MedraWebservice.inc.php

```diff
@@ -149,6 +184,11 @@ function _doRequest($action, $arg, $attachment = null) {
 			if (empty($matches)) {
 				if ($attachment) {
 					assert(PKPString::regexp_match('#<returnCode>success</returnCode>#', $response));
+					if(!assert(PKPString::regexp_match('#<returnCode>success</returnCode>#', $response))){
```

Comment:
I think this is another improper use of `assert`.

Reply:
Change it with empty.